### DISCUSSION TITLE
Changing the keys waiting time

### DISCUSSION THREAD

**Discussion by yuhsienchiang** [nvim-treesitter/nvim-treesitter-textobjects#502] [Q&A]
Recently, I started learning to use motions, and I found treesitter-textobjects.  After trying it out, I noticed that there is no waiting time for the last key in each keymap set by treesitter-textobjects is very short.  

For example, 
nvim native motion keys: `va"` (select "" text object including the quotes)
I don't need to hit `"` right after `a`.  I don't know if there is a timeout, but the motion still works after waiting a while.

treesitter-textobject: `va=` (`["a="]` are set to `@assignment.outer`)
I'll need to hit `=` immediately after `a` to perform the selection.  

I guess this is fine for people who are already familiar with motions. However, is there a way to increase the key waiting time to give beginners like me more time to find the right keys to hit?

**Comment by gregorias** [comment]
Yes, you need to increase `timeoutlen`.
